project/zadanie_domowe/ex4.py

```diff
@@ -0,0 +1,23 @@
+# 4. oblicz ocenę na podstawie progu procentowego
+
+zdobyte_punkty = int(input("Ile zdobyłeś procet: "))
+if zdobyte_punkty > 95 and zdobyte_punkty < 101:
```

Comment:
tutaj lepiej wyglądałby przedział <= 100 jak patrzę na 101 to pierwsza myśl. dlaczego przedział jest do 101, dopiero potem patrzę na to jaki operator porónania jest i oddycham z ulgą ;)